vms/registration/tests/test_functional_volunteer.py

```diff
@@ -272,32 +265,25 @@ def test_location_fields(self):
         page.fill_registration_form(entry)
 
         self.assertNotEqual(page.get_help_blocks(), None)
-        self.assertEqual(
-            self.driver.current_url,
-            self.live_server_url + page.volunteer_registration_page)
+        self.assertEqual(page.remove_i18n(self.driver.current_url),
+                         self.live_server_url + page.volunteer_registration_page)
 
         # verify that messages are displayed for all fields
-        self.assertEqual(page.get_address_error_text(), 'Enter a valid value.')
-        self.assertEqual(page.get_city_error_text(), 'Enter a valid value.')
-        self.assertEqual(page.get_state_error_text(), 'Enter a valid value.')
-        self.assertEqual(page.get_country_error_text(), 'Enter a valid value.')
-
-        # database check to verify that no user, volunteer created
-        self.assertEqual(len(User.objects.all()), 0)
-        self.assertEqual(len(Volunteer.objects.all()), 0)
+        self.assertEqual(page.get_address_error_text(), page.ENTER_VALID_VALUE)
+        self.assertEqual(page.get_city_error_text(), page.ENTER_VALID_VALUE)
+        self.assertEqual(page.get_state_error_text(), page.ENTER_VALID_VALUE)
+        self.assertEqual(page.get_country_error_text(), page.ENTER_VALID_VALUE)
 
     def test_email_field(self):
-
         page = self.page
         page.live_server_url = self.live_server_url
-        # register valid volunteer user
         page.register_valid_details()
 
         # verify successful registration
         self.assertNotEqual(page.get_message_box(), None)
-        self.assertEqual(page.get_message_box_text(), page.success_message)
-        self.assertEqual(self.driver.current_url,
-                         self.live_server_url + PageUrls.homepage)
+        self.assertEqual(page.get_message_box_text(), page.confirm_email_message)
+        self.assertEqual(page.remove_i18n(self.driver.current_url),
+                         self.live_server_url + PageUrls.activate_page)
```

Comment:
Hey @anjali-dhanuka, the user is not redirected to `registration/signup_volunteer/activate/` after the form is submitted.

![image](https://user-images.githubusercontent.com/24565261/42136629-e9f664cc-7d7c-11e8-8767-a20c3c6d6c33.png)